src/mango/src/mango_cursor.erl

```diff
@@ -150,12 +151,53 @@ group_indexes_by_type(Indexes) ->
     end, ?CURSOR_MODULES).
 
 
-maybe_add_warning(UserFun, #idx{type = IndexType}, UserAcc) ->
-    case IndexType of
+maybe_add_warning(UserFun, #cursor{index = Index, opts = Opts}, UserAcc) ->
+
```

Comment:
Is this whitespace intended? I don't think I've seen this before. Might want to check styling guidelines.